### PR TITLE
feat(connectors): add google sheets connector docs

### PR DESCRIPTION
**Pull request by mrebrii**
## Description

Add documentation for a new Google Sheets connector.

## When should this change go live?

June 13

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer.

### CODE REVIEW

**Review by igpetrov — APPROVED** [review]
Looking good from tech side. Very well detailed. Well done!

**Review by bastiankoerber — APPROVED** [review]
really nice documentation. Just had one remark.

**Review by bastiankoerber — APPROVED** [review]
really nice documentation. Just had one remark.

**Review by igpetrov — APPROVED** [review]

**Review by bastiankoerber — APPROVED** [review]

File: docs/components/connectors/out-of-the-box-connectors/available-connectors-overview.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ import TabItem from "@theme/TabItem";
 - [GitLab Connector](/components/connectors/out-of-the-box-connectors/gitlab.md) - Manage [GitLab](https://about.gitlab.com/) issues and releases from your BPMN process.
 - [Google Drive Connector](/components/connectors/out-of-the-box-connectors/googledrive.md) - Create folders or files from a [Google Drive](https://www.google.com/drive/) template from your BPMN process.
 - [Google Maps Platform Connector](/components/connectors/out-of-the-box-connectors/google-maps-platform.md) - Validate addresses, retrieve postal addresses, and calculate distances with [Google Maps Platform Service](https://mapsplatform.google.com/) from your BPMN process
+- [Google Sheets Connector](docs/components/connectors/out-of-the-box-connectors/google-sheets.md) - Create empty spreadsheet on [Google Drive](https://drive.google.com/) and work with it or already existing one from your BPMN process.
```
Comment (akeller) [inline docs/components/connectors/out-of-the-box-connectors/available-connectors-overview.md:43]:
```suggestion
- [Google Sheets Connector](docs/components/connectors/out-of-the-box-connectors/google-sheets.md) - Work with a new or empty spreadsheet on [Google Drive](https://drive.google.com/) from your BPMN process.
```

File: docs/components/connectors/out-of-the-box-connectors/google-sheets.md [hunk 2]
```diff
@@ -0,0 +1,384 @@
+---
+id: google-sheets
+title: Google Sheets Connector
+sidebar_label: Google Sheets Connector
+description: Use the Google Sheets Connector to connect your BPMN service with Google Sheets, and work with spreadsheets using this service.
+---
+
+The **Google Sheets Connector** is an outbound Connector that allows you to create empty spreadsheet
+on [Google Drive](https://drive.google.com/) and work with it or already existing one from your BPMN process.
```
Comment (akeller) [inline docs/components/connectors/out-of-the-box-connectors/google-sheets.md:9]:
```suggestion
The **Google Sheets Connector** is an outbound Connector that allows you to work with an existing or new empty spreadsheet
on [Google Drive](https://drive.google.com/) from your BPMN process.
```

**Comment by akeller** [comment]
I left a few review comments but nothing major.

File: docs/components/connectors/out-of-the-box-connectors/available-connectors-overview.md (hunk 1 above)
Comment (mrebrii) [reply docs/components/connectors/out-of-the-box-connectors/available-connectors-overview.md:43]: changed

File: docs/components/connectors/out-of-the-box-connectors/google-sheets.md [hunk 3]
```diff
@@ -0,0 +1,384 @@
+---
+id: google-sheets
+title: Google Sheets Connector
+sidebar_label: Google Sheets Connector
+description: Use the Google Sheets Connector to connect your BPMN service with Google Sheets, and work with spreadsheets using this service.
```
[the remark this answers is not in the document]
Comment (mrebrii) [reply docs/components/connectors/out-of-the-box-connectors/google-sheets.md:5]: changed

File: docs/components/connectors/out-of-the-box-connectors/google-sheets.md (hunk 2 above)
Comment (mrebrii) [reply docs/components/connectors/out-of-the-box-connectors/google-sheets.md:9]: changed

**Review by igpetrov — DISMISSED** [review]

File: docs/components/connectors/out-of-the-box-connectors/available-connectors-overview.md [hunk 4]
```diff
@@ -40,6 +40,7 @@ import TabItem from "@theme/TabItem";
 - [GitLab Connector](/components/connectors/out-of-the-box-connectors/gitlab.md) - Manage [GitLab](https://about.gitlab.com/) issues and releases from your BPMN process.
 - [Google Drive Connector](/components/connectors/out-of-the-box-connectors/googledrive.md) - Create folders or files from a [Google Drive](https://www.google.com/drive/) template from your BPMN process.
 - [Google Maps Platform Connector](/components/connectors/out-of-the-box-connectors/google-maps-platform.md) - Validate addresses, retrieve postal addresses, and calculate distances with [Google Maps Platform Service](https://mapsplatform.google.com/) from your BPMN process
+- [Google Sheets Connector](docs/components/connectors/out-of-the-box-connectors/google-sheets.md) - allows you to work with an existing or new empty spreadsheet on [Google Drive](https://drive.google.com/) from your BPMN process.
```
Comment (akeller) [inline docs/components/connectors/out-of-the-box-connectors/available-connectors-overview.md:43]:
```suggestion
- [Google Sheets Connector](/components/connectors/out-of-the-box-connectors/google-sheets.md) - allows you to work with an existing or new empty spreadsheet on [Google Drive](https://drive.google.com/) from your BPMN process.
```

File: versioned_docs/version-8.2/components/connectors/out-of-the-box-connectors/available-connectors-overview.md [hunk 5]
```diff
@@ -40,6 +40,7 @@ import TabItem from "@theme/TabItem";
 - [GitLab Connector](/components/connectors/out-of-the-box-connectors/gitlab.md) - Manage [GitLab](https://about.gitlab.com/) issues and releases from your BPMN process.
 - [Google Drive Connector](/components/connectors/out-of-the-box-connectors/googledrive.md) - Create folders or files from a [Google Drive](https://www.google.com/drive/) template from your BPMN process.
 - [Google Maps Platform Connector](/components/connectors/out-of-the-box-connectors/google-maps-platform.md) - Validate addresses, retrieve postal addresses, and calculate distances with [Google Maps Platform Service](https://mapsplatform.google.com/) from your BPMN process
+- [Google Sheets Connector](docs/components/connectors/out-of-the-box-connectors/google-sheets.md) - allows you to work with an existing or new empty spreadsheet on [Google Drive](https://drive.google.com/) from your BPMN process.
```
Comment (akeller) [inline versioned_docs/version-8.2/components/connectors/out-of-the-box-connectors/available-connectors-overview.md:43]:
```suggestion
- [Google Sheets Connector](/components/connectors/out-of-the-box-connectors/google-sheets.md) - allows you to work with an existing or new empty spreadsheet on [Google Drive](https://drive.google.com/) from your BPMN process.
```

File: versioned_docs/version-8.1/components/connectors/out-of-the-box-connectors/available-connectors-overview.md [hunk 6]
```diff
@@ -40,6 +40,7 @@ import TabItem from "@theme/TabItem";
 - [GitLab Connector](/components/connectors/out-of-the-box-connectors/gitlab.md) - Manage [GitLab](https://about.gitlab.com/) issues and releases from your BPMN process.
 - [Google Drive Connector](/components/connectors/out-of-the-box-connectors/googledrive.md) - Create folders or files from a [Google Drive](https://www.google.com/drive/) template from your BPMN process.
 - [Google Maps Platform Connector](/components/connectors/out-of-the-box-connectors/google-maps-platform.md) - Validate addresses, retrieve postal addresses, and calculate distances with [Google Maps Platform Service](https://mapsplatform.google.com/) from your BPMN process
+- [Google Sheets Connector](docs/components/connectors/out-of-the-box-connectors/google-sheets.md) - allows you to work with an existing or new empty spreadsheet on [Google Drive](https://drive.google.com/) from your BPMN process.
```
Comment (akeller) [inline versioned_docs/version-8.1/components/connectors/out-of-the-box-connectors/available-connectors-overview.md:43]:
```suggestion
- [Google Sheets Connector](/components/connectors/out-of-the-box-connectors/google-sheets.md) - allows you to work with an existing or new empty spreadsheet on [Google Drive](https://drive.google.com/) from your BPMN process.
```

File: versioned_docs/version-8.0/components/connectors/out-of-the-box-connectors/available-connectors-overview.md [hunk 7]
```diff
@@ -39,6 +39,7 @@ import TabItem from "@theme/TabItem";
 - [GitLab Connector](/components/connectors/out-of-the-box-connectors/gitlab.md) - Manage [GitLab](https://about.gitlab.com/) issues and releases from your BPMN process.
 - [Google Drive Connector](/components/connectors/out-of-the-box-connectors/googledrive.md) - Create folders or files from a [Google Drive](https://www.google.com/drive/) template from your BPMN process.
 - [Google Maps Platform Connector](/components/connectors/out-of-the-box-connectors/google-maps-platform.md) - Validate addresses, retrieve postal addresses, and calculate distances with [Google Maps Platform Service](https://mapsplatform.google.com/) from your BPMN process
+- [Google Sheets Connector](docs/components/connectors/out-of-the-box-connectors/google-sheets.md) - allows you to work with an existing or new empty spreadsheet on [Google Drive](https://drive.google.com/) from your BPMN process.
```
Comment (akeller) [inline versioned_docs/version-8.0/components/connectors/out-of-the-box-connectors/available-connectors-overview.md:42]:
```suggestion
- [Google Sheets Connector](/components/connectors/out-of-the-box-connectors/google-sheets.md) - allows you to work with an existing or new empty spreadsheet on [Google Drive](https://drive.google.com/) from your BPMN process.
```

**Pull request merged**